index.ts

```diff
@@ -118,10 +117,10 @@ const run = async (): Promise<void> => {
       await updateComment({ context, octokit, commentId, commentText });
     }
 
-    if (!sync && !githubPersonalAccessToken && octokit) {
+    if (!sync && octokit) {
       await createComment({
         commentText:
-          'Unable to update test status any further, please include a Github token or sync argument',
+          'Unable to update check status any further, please include a Github PAT and sync argument',
```

Comment:
I think this is an 'or'. We can theoretically do async with a PAT or sync with default action token